AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginGraphQLUserPoolTests/Base/AsyncExpectation.swift

```diff
@@ -0,0 +1,196 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import XCTest
+import Amplify
+
+public actor AsyncExpectation {
```

Comment:
Because the HostApp (DataStoreHostApp, APIHostApp, etc) do not have access to AmplifyTestCommon, I am currently copying the AsyncExpectation implementation local to the test target. Had chatted with @ameter before about this but we haven't figured out a better path forward. I could also put it inside DataStoreHostApp app target and then all targets will get it with the `import DataStoreHostApp` too